crates/types/src/errors.rs

```diff
@@ -78,6 +78,7 @@ pub mod codes {
     pub const GONE: InvocationErrorCode = InvocationErrorCode(410);
     pub const JOURNAL_MISMATCH: InvocationErrorCode = InvocationErrorCode(570);
     pub const PROTOCOL_VIOLATION: InvocationErrorCode = InvocationErrorCode(571);
+    pub const CONFLICT: InvocationErrorCode = InvocationErrorCode(409);
```

Comment:
Maybe add an explanation what a conflict constitutes (e.g. trying to complete an already completed promise).

Reply:
CONFLICT is a generic HTTP status code, i added some details to the error itself.